src/persistence/profile.h

```diff
@@ -98,7 +98,6 @@ private slots:
     bool newProfile;
     bool isRemoved;
     static QVector<QString> profiles;
-    static constexpr int encryptHeaderSize = 8;
```

Comment:
I'm nut sure, but seems `TOX_PASS_ENCRYPTION_EXTRA_LENGTH == 80`, but `encryptHeaderSize == 8`. It's correct?